packages/teams-js/src/public/dialog.ts

```diff
@@ -249,7 +249,7 @@ export namespace dialog {
      */
     export namespace bot {
       /**
-       * Allows an app to open the dialog module using bot.
+       * Allows an app to open the dialog module that sends the submit data to the bot.
```

Comment:
```suggestion
       * Allows an app to open the dialog module that sends the submit data to a bot.
```

Reply:
Also consider "Allows an app to open a dialog that sends submitted data to a bot"

Reply:
Thank you for the suggestions! ❤️